README.md

```diff
@@ -72,94 +72,111 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
-### Keyboard Manager
+### PowerRename
 
-- Adjusted wording on the editor when keys are orphaned.
+- Fixed an issue that was generating a silent crash when the context menu was triggered when not selecting any file or folder. (This was a hotfix for 0.61)
+- Improved performance when loading a big number of files.
+- Fixed a specific case in which PowerRename tried to rename a file to an empty string.
+- The UI now shows when a file can't be renamed due to its name being too long or containing invalid characters.
 
-### Mouse utility
+### PowerToys Run
 
-- Fixed a bug that caused the current Find My Mouse spotlight to hang when activated in the top left corner of the screen. (This was a hotfix for 0.60)
+- Added a fix to the VSCodeWorkspaces plugin to better support portable installations. Thanks [@bkmeneguello](https://github.com/bkmeneguello)!
+- The Folder plugin now expands `%HOMEPATH%` correctly.
+- Fixed a case where a previous result was being activated when searching for new results. Added a setting to better control input throttling. Thanks [@jefflord](https://github.com/jefflord)!
+- Added support for port numbers in the URI plugin. Thanks [@KohGeek](https://github.com/KohGeek)!
+- Fixed query errors when the search delay option was turned off.
+- New History plugin to search for old search results. Thanks [@jefflord](https://github.com/jefflord)!
+- Changed the default TimeDate activation keyword to `)` and added information in Settings so users know that some activation keywords may conflict with normal usage of some plugins when trying to do a global query. Thanks [@htcfreek](https://github.com/htcfreek)!
+- The Unit Converter plugin updated its UnitsNet dependency and now supports plural units. Thanks [@FWest98](https://github.com/FWest98)!
+- The Calculator plugin improved its input validation logic. Thanks [@htcfreek](https://github.com/htcfreek)!
 
-### PowerRename
+### Runner
 
-- The PowerRename window reacts to current dpi when created.
+- Clean up old install folders and logs at startup. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
 
-### PowerToys Run
+### Settings
 
-- Fixed a typo in the WindowWalker plugin UI. Thanks [@rohanrdy](https://github.com/rohanrdy)!
-- Improved performance by saving the search history files only on exit. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
-- PowerToys Run no longer shows results for some plugins when querying for empty spaces in a global query. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
-- Added support for showing localized names for some win32 programs in the programs plugin. Thanks [@htcfreek](https://github.com/htcfreek)!
-- The program plugin will now consider settings changed directly in ProgramPluginSettings.json. Thanks [@bezgumption](https://github.com/bezgumption)!
+- Image and phrasing adjustments. Thanks [@htcfreek](https://github.com/htcfreek)!
```

Comment:
what changes should that be? 🤔

Reply:
And what's holding my pr? I was hoping it would be accepted by now?

Reply:
I added it from here, since I saw your name in the commit : https://github.com/microsoft/PowerToys/commit/55c5cf850fc8f7be19b82a7e73386ac6aa3ab6ad
It was just a typo, though, and was mostly crutkas. I'll remove it 😅

Reply:
@Jay-o-Way, it's waiting on niels review. I assume would be good for 0.63.